### PR TITLE
add csv file output and bug fixed for the output save

### PR DESCRIPTION
**Pull request by clarafischer**
I add a csv output file like the html which can be easily parse and have a rapid view of which subject did not finish.
Plus i correct a bug in a list which appear with python3 i think.

### CODE REVIEW

File: brainvisa/toolboxes/data management/processes/database_qc_table.py [hunk 1]
```diff
@@ -757,7 +759,7 @@ def save_html(self, filename, context=None):
                         for row, row_id in six.iteritems(rev_row_ids)])
 
     # sort items
-    rows_order = zip(*sorted(zip(list(rev_row_ids.values()), list(range(nrows)))))[1]
+    rows_order = list(zip(*sorted(zip(rev_row_ids.values(), range(nrows)))))[1]
```
Comment (clarafischer) [inline brainvisa/toolboxes/data management/processes/database_qc_table.py:763]: Bug fixed, I think it was not working this python3.

File: brainvisa/toolboxes/data management/processes/database_qc_table.py [hunk 2]
```diff
@@ -24,7 +24,8 @@
     'data_filters', ListOf(String()),
     'keys', ListOf(String()),
     'type_labels', ListOf(String()),
-    'output_file', WriteDiskItem('Text File', 'HTML PDF'),
+    'output_file', WriteDiskItem(
```
Comment (clarafischer) [inline brainvisa/toolboxes/data management/processes/database_qc_table.py:27]: How the format 'HTML PDF' was working, or I did not find were it was declare may be ?

Comment (denisri) [reply brainvisa/toolboxes/data management/processes/database_qc_table.py:27]: It'sa "formats list" defined in brainvisa.tools.aimsGlobals which contains the HTML and optionally the PDF format only if the "wkhtmltopdf" program is found (html top pdf converter).
I suppose we can use `['HTML PDF', 'CSV file']`

File: brainvisa/toolboxes/data management/processes/database_qc_table.py (hunk 1 above)
Comment (denisri) [reply brainvisa/toolboxes/data management/processes/database_qc_table.py:763]: probably right ;)

**Review by denisri — APPROVED** [review]
OK for me.

File: brainvisa/toolboxes/data management/processes/database_qc_table.py (hunk 2 above)
Comment (clarafischer) [reply brainvisa/toolboxes/data management/processes/database_qc_table.py:27]: Ah yes like 'aims writeable volume formats'. I tried `['HTML PDF', 'CSV file']` but it does not work, it say :
`'HTML PDF' n'est pas un format valide`.

Comment (denisri) [reply brainvisa/toolboxes/data management/processes/database_qc_table.py:27]: Ah crap, it's not recursive.
Then try `getFormats('HTML PDF') + ['CSV file']`

**Pull request merged**